Sources/TootSDK/TootClient/TootClient.swift

```diff
@@ -28,14 +28,14 @@ public class TootClient: @unchecked Sendable {
     public lazy var data = TootDataStream(client: self)
     /// The clientName the client was initialized with
     public let clientName: String
+    /// The current accessToken in use
+    public internal(set) var accessToken: String?
```

Comment:
Honestly, I don't think this is a good route to go. The consuming app has responsibility for persisting the token (in the keychain please!) and giving it to the TootClient, so you should already have the current token stored securely and shouldn't need to read it back from the client.

One of the reasons the accessToken is pushed down and made internal, is also to protect it from being easily read by any 3rd party services outside of TootSDK. It is an edge-case, but it's one we don't have to expose.

TLDR: I don't mind the isAnonymous flag, but I'd rather we leave this property internal to the SDK.

Reply:
Reverted changes leaving only `isAnonymous`.